Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,742 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Threat Model
+============
+
+This section presents a threat model for Cilium. This threat model
+allows interested parties to understand:
+
+-  security-specific implications of Cilium’s architecture
+-  controls that are in place to secure data flowing through Cilium’s various components
+-  recommended controls for running Cilium in a production environment
+
+Scope and Prerequisites
+-----------------------
+
+This threat model considers the possible attacks that could affect an
+up-to-date version of Cilium running in a production environment; it
+will be refreshed when there are significant changes to Cilium’s
+architecture or security posture.
+
+This model does not consider supply-chain attacks, such as attacks where
+a malicious contributor is able to intentionally inject vulnerable code
+into Cilium. For users who are concerned about supply-chain attacks,
+Cilium’s `security audit`_ assessed Cilium’s supply chain controls
+against `the SLSA framework`_.
+
+In order to understand the following threat model, readers will need
+familiarity with basic Kubernetes concepts, as well as a high-level
+understanding of Cilium’s :ref:`architecture and components<component_overview>`.
+
+.. _recent security audit: https://github.com/cilium/cilium.io/blob/main/Security-Reports/CiliumSecurityAudit2022.pdf
```

Comment:
This has not been updated, causing the GitHub workflow to complain:

```suggestion
.. _security audit: https://github.com/cilium/cilium.io/blob/main/Security-Reports/CiliumSecurityAudit2022.pdf
```

Reply:
Thanks for the spot!